k8s/models/pod.py

```diff
@@ -130,6 +130,7 @@ class Container(Model):
     livenessProbe = Field(Probe)
     readinessProbe = Field(Probe)
     imagePullPolicy = Field(six.text_type, "IfNotPresent")
+    command = Field(list)
```

Comment:
This needs to be `ListField(six.text_type)`, not `Field(list)`.

Reply:
It's list to support arguments. 

        container = Container(
            name="pi",
            image="perl",
            command=["perl", "-Mbignum=bpi", "-wle", "print bpi(2000)"]
        )

see https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.9/#create-38

Reply:
The argument is a string array in the API-reference. Arrays in the API should be represented by a `ListField`, which handles some details of how arrays works. Each value in the array is a string, so the type is `six.text_type`. Using `Field(list)` will work in many cases, but there are some edge cases where it breaks. You also lose the type-support, since a python list can contain values of any type, while the k8s API only accepts strings in this case.

Reply:
Sorry, misread your previous comment. I've changed it.